site/app/templates/misc/MarkdownArea.twig

```diff
@@ -109,14 +109,37 @@
     </div>
 </div>
 <script>
-    $('#{{markdown_area_id}}').ready(function() {
-        enableTabsInTextArea("#{{markdown_area_id}}");
-        const markdown_area = $('#{{markdown_area_id}}').closest('.markdown-area')
-        markdown_area.find('[data-initialize-preview="1"]').trigger('click');
+$(document).ready(function() {
+    function resizeTextarea(textarea) {
```

Comment:
I would make this an event listener to apply to all textarea's on the forum thread for future updates.